src/components/App/index.js

```diff
@@ -155,11 +157,75 @@ export class App extends Component {
 		I18n.on('change', this.handleLanguageChange);
 	}
 
+	locationBackup = async() => {
+		const { ip } = await fetch(`${ 'https://cors-anywhere.herokuapp.com/' }https://api.ipify.org?format=json`, {
+			mode: 'cors',
+			headers: {
+				'Access-Control-Allow-Origin': '*',
+			},
+		}).then((res) => (res.json()));
+
+		const location = await fetch(`${ 'https://cors-anywhere.herokuapp.com/' }https://geoip-db.com/json/${ ip }`, {
```

Comment:
We need to define the service which will provide the geolocation data on the server side.
Hardcoding is not a good approach.

Reply:
Ok, let me do that